django/contrib/postgres/constraints.py

```diff
@@ -206,9 +206,16 @@ def validate(self, model, instance, exclude=None, using=DEFAULT_DB_ALIAS):
                     self.get_violation_error_message(), code=self.violation_error_code
                 )
         else:
-            if (self.condition & Exists(queryset.filter(self.condition))).check(
-                replacement_map, using=using
-            ):
-                raise ValidationError(
-                    self.get_violation_error_message(), code=self.violation_error_code
-                )
+            against = instance._get_field_expression_map(
+                meta=model._meta, exclude=exclude
+            )
```

Comment:
This is exactly the same as `replacement_map` so there is no need to reevaluate it.

Reply:
I had referred to `django/db/models/constraints.py` and adopted it, but upon reviewing above, I noticed there's already an identical part. Thank you!